cli/tests/integration_tests.rs

```diff
@@ -140,13 +140,6 @@ fn send_mined_sapling_to_orchard() {
         let balance = client.do_balance().await;
         // We send change to orchard now, so we should have the full value of the note
         // we spent, minus the transaction fee
-        assert_eq!(
-            balance["unverified_orchard_balance"],
-            625_000_000 - u64::from(DEFAULT_FEE)
-        );
-        assert_eq!(balance["verified_orchard_balance"], 0);
-        utils::increase_height_and_sync_client(&regtest_manager, &client, 1).await;
-        let balance = client.do_balance().await;
```

Comment:
Why are these `assert_eq!` removed?

Reply:
We were asserting that the funds where unverified, then syncing higher, then asserting that they're verified. Now they're verified immediately, so there's no need for the second step.

Reply:
Hmmm...    they're not really "verified"...   we're just not waiting for verification to send?   I mean...   isn't there still an idea of something that's more certain?